tests/zenko_tests/node_tests/s3SDK.js

```diff
@@ -33,4 +33,18 @@ const awsS3Client = new S3({
     httpOptions: { timeout: 0 },
 });
 
-module.exports = { scalityS3Client, awsS3Client };
+const ringS3Client = new S3([
+    accessKeyId: process.env.RING_S3C_ACCESS_KEY,
```

Comment:
I just added the secrets to the CI backend. These env vars will still need to be added to your eve/main.yml in `RING_S3C_ACCESS_KEY: '%(secret:ring_s3c_access_key)s'` format here:
https://github.com/scality/Zenko/blob/development/1.1/eve/main.yml#L69 

And add the env vars here as well:
https://github.com/scality/Zenko/blob/development/1.1/eve/workers/ci_env.sh#L21

Once you have that the tests should be able to grab the appropriate secrets from the env.

Reply:
Just to double check - since the other `env` values in `main.yml` seem to take the format of `'%(secret:<location>_backend_access_key)s'`,  should I follow that format? and use
`'%(secret:ring_s3c_backend_access_key)s'`
`'%(secret:ring_s3c_backend_secret_key)s'`

Reply:
Yeah sorry that was my bad. Let's keep it consistent.

Reply:
np, ty for the info :D